packages/serverpod/lib/src/server/websocket_request_handlers/helpers/method_stream_manager.dart

```diff
@@ -336,9 +340,12 @@ class MethodStreamManager {
       /// or a request from the client.
       if (isCancelled) return;
       isCancelled = true;
+      session.serverpod.logVerbose(
```

Comment:
Question/Suggestion: Should we log these for the user with the debug log level?

Since this "action" is associated with a specific user it would make sense to track the log together with the other user logs.